app/models/rbac.rb

```diff
@@ -271,7 +262,7 @@ def self.find_options_for_tenant(klass, user_or_group, find_options = {})
     tenant_ids = klass.accessible_tenant_ids(user_or_group, accessible_tenant_ids_strategy(klass))
     return find_options if tenant_ids.empty?
 
-    tenant_id_clause = {klass.table_name => {:tenant_id => tenant_ids}}
+    tenant_id_clause = ["#{klass.table_name}.tenant_id IN (?)", tenant_ids]
```

Comment:
Not with manual sanitize_sql. Once we manage to scope-ify everything here, we can switch back to the much better-looking hash.